packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-table.jsx

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, Text } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+
+const BarriersTable = ({
+    barrier_1,
+    setChosenBarrier,
+    is_barriers_table_expanded,
+    turbos_barrier_choices,
+    toggleBarriersTable,
+}) => {
+    const [new_barrier, setNewBarrier] = useState(barrier_1);
+    const changeBarrier = e => {
+        e.stopPropagation();
+        setNewBarrier(e.target.innerText);
+        setChosenBarrier(e.target.innerText);
+    };
+    //TODO: remove slice after adding scroll
+    const barriers_list = turbos_barrier_choices.slice(0, 16).map(barrier => (
+        <li
+            key={barrier}
+            className={`trade-container__barriers-table__item ${
+                new_barrier === barrier ? 'trade-container__barriers-table__item--selected' : ''
+            }`}
+            onClick={changeBarrier}
```

Comment:
I put `onClick` function not on `ul` but on each `li` because in other case we need to check did the user click on `li` and on which exactly.

Reply:
@kate-deriv please use our existing reusable components from components package instead of native ones)

Reply:
Yes, I'll replace current components with the existing one today

Reply:
I checked all existing components, but each of them should be radically refactor if we want to reuse them for turbos (for e.g. DataList, DataTable, Table, VerticalTab, ListItem and etc) . Moreover, it'll make them too heavy and might case decomposition, that is why I suggest to create a new simple and highly customized component for list of barriers option.